src/System.Numerics.Vectors/tests/System.Numerics.Vectors.Tests.csproj

```diff
@@ -0,0 +1,65 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>$(NetFrameworkMinimum)</TargetFramework>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <EnableDefaultItems>false</EnableDefaultItems>
```

Comment:
Not blocking - but why did we disable default items?  Looks to me like they could work (maybe with Update item for GenericVectorTests.cs to add the metadata).

Reply:
It was getting complicated to condition the item groups differently.
I can address it in a separate PR: https://github.com/dotnet/maintenance-packages/issues/66